### PR TITLE
Header hardening

### PR DESCRIPTION
**Pull request by tkohr**
This PR should resolve:

- #4 => replacing url with demo instance (only used for dev)
- #5 => getting lang param from browser or web component attribute
- #7 => add attribute to indicate logo url and embed georchestra logo svg as fallback
- #10 => make header responsive and hide admin menu on mobile 
- #12 
- #13

### CODE REVIEW

**Comment by f-necas** [comment]
I've got an unexpected behaviour when putting it  with georchestra, height/padding's broken: 
Peek 2023-11-06 12-21

**Comment by f-necas** [comment]
- Added some design improvements in order to have only a 'one line' header instead of two. 
- Responsiveness has also been improved to be able to match different heights.
- Cleaned inkscape's attributes in Georchestra's SVG.

Peek 2023-11-07 14-21

For review : @tkohr

**Comment by f-necas** [comment]
I have open two more issues to improve header. Do we add them there or in another PR ?

**Comment by tkohr** [comment]
> I have open two more issues to improve header. Do we add them there or in another PR ?

If you are working on it, I'd say don't hesitate to push it in this PR. Otherwise, we'll do separate PRs. I can do a review/testing today, I think.

File: src/header.ce.vue [hunk 1]
```diff
@@ -3,17 +3,27 @@ import { computed, onMounted, reactive } from 'vue'
...
+import ChartPieIcon from '@/ui/ChartPieIcon.vue'
+import UsersIcon from '@/ui/UsersIcon.vue'
+import ChevronDownIcon from '@/ui/ChevronDownIcon.vue'
 
 const props = defineProps<{
   lang?: string
   activeApp?: string
+  logoUrl?: string
   //legacy option : using old iframe option
+  legacyHeader: string
```
Comment (tkohr) [inline src/header.ce.vue:18]: Shouldn't this be optional?

File: src/header.ce.vue [hunk 2]
```diff
@@ -27,75 +37,231 @@ const loginUrl = computed(() => {
...
+    'eng'
   getUserDetails().then(user => {
     state.user = user
   })
 })
 </script>
 <template>
-  <div v-if="props.legacyUrl">
+  <!-- props.legacyHeader -->
+  <div v-if="'true' == props.legacyHeader">
```
Comment (tkohr) [inline src/header.ce.vue:73]:
```suggestion
  <div v-if="props.legacyHeader === 'true'">
```

File: src/header.ce.vue [hunk 3]
```diff
@@ -27,75 +37,231 @@ const loginUrl = computed(() => {
...
     <iframe
-      v-bind:src="props.legacyUrl"
-      v-bind:style="props.legacyStyle"
+      v-bind:src="
+        props.legacyUrl + (props.activeApp ? '?active=' + props.activeApp : '')
+      "
+      v-bind:style="props.style"
```
Comment (tkohr) [inline src/header.ce.vue:78]:
```suggestion
      :src="`${props.legacyUrl}${
        props.activeApp ? `?active=${props.activeApp}` : ''
      }`"
      :style="props.style"
```

File: src/header.ce.vue [hunk 4]
```diff
@@ -27,75 +37,231 @@ const loginUrl = computed(() => {
...
-        class="console py-1 hover:text-secondary/60"
-        >console</a
-      >
-      <a href="/analytics/" class="console py-1 hover:text-secondary/60"
-        >analytics</a
-      >
-    </div>
-    <div class="h-20 flex justify-between text-slate-600">
+  <header v-else class="host h-full" v-bind:style="props.style">
+    <div class="justify-between text-slate-600 sm:flex hidden h-full">
```
Comment (tkohr) [inline src/header.ce.vue:82]: I think this got lost:
```suggestion
    <div class="justify-between bg-white text-slate-600 sm:flex hidden h-full">
```

**Review by tkohr — COMMENTED** [review]
Thanks, @f-necas, really nice improvements! I'm just not a big fan of the icons in the admin dropdown, don't have a better proposition though.

I've left some minor suggestions. Also did some testing with the datahub and console within georchestra, but not extensively yet.

**Comment by tkohr** [comment]
Thanks for the translations @f-necas ! Strangely, when displaying the header in analytics, I get an error since the addition of i18n. Can you reproduce this on your side ?
analytics
I just added a commit for the icon alignment.

**Review by tkohr — COMMENTED** [review]
Thanks for the continued effort and the latest fixes here (in particular the build format). Everything works fine on my side. I just can't officially approve the PR.

**Pull request merged**